extensions/amp-dynamic-css-classes/amp-dynamic-css-classes.md

```diff
@@ -16,8 +16,38 @@ limitations under the License.
 
 ### <a name="amp-dynamic-css-classes"></a> AMP Dynamic CSS Classes
 
-The AMP Dynamic CSS Classes extension adds several dynamic CSS class
-names onto the HTML element.
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Adds several dynamic CSS class names onto the HTML element.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td><a href="/docs/reference/experimental.html">Experimental</a></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-dynamic-css-classes" src="https://cdn.ampproject.org/v0/amp-dynamic-css-classes-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td>None</td>
+  </tr>
+</table>
+
+The following lists validation errors specific to the `amp-dynamic-css-classes` tag
```

Comment:
Unlike the other custom-elements, this script doesn't actually introduce a new tag. It's perhaps poorly named as a custom-element. Specifically, there is no "`amp-dynamic-css-classes` tag". Thus REQUIRED_SCRIPT is also not a possible error.